### PR TITLE
Correctly fix dbt2 db connection controls

### PR DESCRIPTION
**Pull request by mw2q**
Finally correctly fix how database connections are controled.  c09318c460da2b804b4230de08e5ba8f6a7d0cc1 and 75ecaa3c9241e8334bd98462b64cc507e80a2837 were incorrect attempts that have been reverted.

### CODE REVIEW

File: aws-dbt2-biganimal/execute/playbook-dbt2-run.yml [hunk 1]
```diff
@@ -54,11 +63,12 @@
             -a pgsql
             -d {{ dbt2_duration }}
             -D {{ infra.servers.databases.dbt2.dbname }}
+            -F {{ ((dbt2_connections | int) / ansible_processor_nproc) | int }}
```
Comment (jt-edb) [inline aws-dbt2-biganimal/execute/playbook-dbt2-run.yml:66]: Use `fpp`

File: aws-dbt2-rds/execute/playbook-dbt2-run.yml [hunk 2]
```diff
@@ -49,11 +58,12 @@
             -a pgsql
             -d {{ dbt2_duration }}
             -D {{ infra.servers.databases.dbt2.dbname }}
+            -F {{ ((dbt2_connections | int) / ansible_processor_nproc) | int }}
```
Comment (jt-edb) [inline aws-dbt2-rds/execute/playbook-dbt2-run.yml:61]: Use `fpp`

File: aws-dbt2-biganimal/execute/playbook-dbt2-run.yml (hunk 1 above)
Comment (mw2q) [reply aws-dbt2-biganimal/execute/playbook-dbt2-run.yml:66]: Oops, fixing...

File: aws-dbt2-rds/execute/playbook-dbt2-run.yml (hunk 2 above)
Comment (mw2q) [reply aws-dbt2-rds/execute/playbook-dbt2-run.yml:61]: Oops, fixing...

**Comment by mw2q** [comment]
Rebased with fpp fixes.

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**